frontend/packages/client/jsconfig.json

```diff
@@ -1,7 +1,10 @@
 {
   "compilerOptions": {
-    "baseUrl": "src"
+    "baseUrl": "src",
+    "jsx": "react-jsx"
```

Comment:
there are still some `import React from 'react':` in the components code, but can safely be removed later on a second PR to avoid increasing the size if this one (It's not the purpose of the PR)
<img width="511" alt="Screen Shot 2022-09-14 at 11 35 03" src="https://user-images.githubusercontent.com/7526740/190184839-39dd43f4-b223-4776-aed3-916956966d68.png">